modules/browser/BrowserCapabilities.js

```diff
@@ -37,6 +37,17 @@ export default class BrowserCapabilities extends BrowserDetection {
         return this.isChromiumBased() || this.isWebKitBased() || this.isFirefox();
     }
 
+    /**
+     * Checks if the client is running on an Android browser.
+     *
+     * @returns {boolean}
+     */
+    isAndroidBrowser() {
+        const { userAgent } = navigator;
+
+        return this.isIosBrowser() || (!this.isReactNative() && userAgent.match(/Android/i));
```

Comment:
Why does this return true if it's an iOS browser?

Reply:
Oops, I moved things around and forgot to remove this one, will delete it.